data/src/main/scala/ch.epfl.scala.index.data/github/GithubReader.scala

```diff
@@ -99,8 +103,50 @@ object GithubReader {
     val repoTopicsPath = githubRepoTopicsPath(paths, github)
     val graphqlResult = read[GraphqlResult](
       Files.readAllLines(repoTopicsPath).toArray.mkString(""))
-    graphqlResult.data.repository.repositoryTopics.nodes.map { node =>
-      node.topic.name
+    graphqlResult.data.get.repository.get.repositoryTopics.get.nodes.get.map { node =>
+      node.topic.get.name.get
```

Comment:
Did you check what happens if those fields are empty?

In general, calling `.get` on `Option`s doesn’t help because it tends to make you forget about handling all the states of your application. Try to implement something without any `.get` call.